### PR TITLE
Added Selection_Sort.php

### LINKED ISSUE #2154: Selection Sort
- [x] C
- [x] C++
- [x] C#
- [x] CoffeeScript
- [ ] Dart - @pooja-git11
- [x] Go
- [x] Java
- [x] JS
- [x] Kotlin
- [x] Php - @BhardwajShweta25
- [x] Python
- [x] Readme
- [x] Ruby
- [x] Rust - @raksha009
- [ ] TypeScript - @lakshyajit165

### PR DESCRIPTION
**Pull request by BhardwajShweta25**
Fixes #2154 
#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Languages Used:
- PHP

#### Files Added:
- Selection_Sort.php

### CODE REVIEW

**Comment by BhardwajShweta25** [comment]
Please let me know the improvements, if any.

File: Selection_Sort/Selection_Sort.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            $ar[$min_idx]=$ar[$i];
+            $ar[$i]=$temp;  
+        }  
+
+        // Print the sorted array
+        for($i=0;$i<$size;$i=$i+1)
+            echo "$ar[$i] ";
+    }
+
+// Sample Input
```
Comment (jainaman224) [inline Selection_Sort/Selection_Sort.php]: Indent this as well.
It should align with upper block.

File: Selection_Sort/Selection_Sort.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?php
+    //Selection Sort function
```
Comment (jainaman224) [inline Selection_Sort/Selection_Sort.php]: Add space after `//`.

File: Selection_Sort/Selection_Sort.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<?php
+    //Selection Sort function 
+    function selection_sort($ar,$size)
```
Comment (jainaman224) [inline Selection_Sort/Selection_Sort.php]: Add space after `,`.

File: Selection_Sort/Selection_Sort.php [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+// count() function returns the number of elements in an array
+$size=count($ar);
+
+// function call 
+selection_sort($ar,$size);
+
+?>
+
+// Sample Output
+// 1 3 4 5 10
```
Comment (jainaman224) [inline Selection_Sort/Selection_Sort.php]: Add a single new line at the end of file.

File: Selection_Sort/Selection_Sort.php (hunk 1 above)
Comment (BhardwajShweta25) [reply Selection_Sort/Selection_Sort.php]: Do you mean the for loop to print array? shouldn't that be indented with the same spaces as the i loop above (line 5) only?

File: Selection_Sort/Selection_Sort.php [hunk 5]
```diff
@@ -0,0 +1,35 @@
+<?php
+    // Selection Sort function 
+    function selection_sort($ar, $size)
+    {
+        for ($i = 0; $i < $size-1; $i=$i+1)
```
Comment (SakshiShreya) [inline Selection_Sort/Selection_Sort.php]: Add space around operators.

File: Selection_Sort/Selection_Sort.php [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+            $ar[$i]=$temp;  
+        }  
+
+        // Print the sorted array
+        for($i=0;$i<$size;$i=$i+1)
+            echo "$ar[$i] ";
+    }
+
+    // Sample Input 
+    $ar=array(10,5,4,3,1);
```
Comment (SakshiShreya) [inline Selection_Sort/Selection_Sort.php]: Add space after `,`

Comment (BhardwajShweta25) [reply Selection_Sort/Selection_Sort.php]: I have made the changes, kindly check.
Thank you!

**Comment by somya-kapoor** [comment]
@BhardwajShweta25 Are you working on it?

**Comment by BhardwajShweta25** [comment]
Could you please check the changes?

File: Selection_Sort/Selection_Sort.php [hunk 7]
```diff
@@ -0,0 +1,35 @@
+<?php
```
Comment (singh-shreya6) [inline Selection_Sort/Selection_Sort.php:1]: Please add a brief problem description as multi line comment.

File: Selection_Sort/Selection_Sort.php [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+            $ar[$i] = $temp;  
+        }  
+
+        // Print the sorted array
+        for($i = 0; $i < $size; $i = $i + 1)
+            echo "$ar[$i] ";
+    }
+
+    // Sample Input 
+    $ar=array(10, 5, 4, 3, 1);
```
Comment (singh-shreya6) [inline Selection_Sort/Selection_Sort.php:25]: Please add space around  =

File: Selection_Sort/Selection_Sort.php [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+    // count() function returns the number of elements in an array
+    $size=count($ar);
+
+    // function call 
+    selection_sort($ar, $size);
+
+?>
+// Sample Output
```
Comment (singh-shreya6) [inline Selection_Sort/Selection_Sort.php:34]: Please add this output as multi line comment

File: Selection_Sort/Selection_Sort.php [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+        // Print the sorted array
+        for($i = 0; $i < $size; $i = $i + 1)
+            echo "$ar[$i] ";
+    }
+
+    // Sample Input 
+    $ar=array(10, 5, 4, 3, 1);
+
+    // count() function returns the number of elements in an array
+    $size=count($ar);
```
Comment (singh-shreya6) [inline Selection_Sort/Selection_Sort.php:28]: Please add space around =

**Comment by singh-shreya6** [comment]
@BhardwajShweta25 If your PR is not reviewed for long, please reach out to other mentors, we will review.

**Comment by BhardwajShweta25** [comment]
@singh-shreya6 I have made the said changes. 
Thank you for seeing the issue.

**Comment by BhardwajShweta25** [comment]
> Looks good @BhardwajShweta25. Please squash your commits.

Done, Please Check.
Thank you!

**Pull request merged**